hail/src/main/scala/is/hail/expr/ir/Sym.scala

```diff
@@ -0,0 +1,57 @@
+package is.hail.expr.ir
+
+import is.hail.utils.StringEscapeUtils
+
+// FIXME Sym instead of Symbol so we don't conflict with Scala's Symbol which is currently used by by the IR DSL
```

Comment:
Delete Fixme.

Reply:
I mean, I want to name it Symbol but I can't rename it until the IR DSL is using Sym instead of Scala's Symbol.